greenkeeper.json

```diff
@@ -0,0 +1,10 @@
+{
+  "groups": {
+    "default": {
+      "packages": [
+        "concourse/create-test-plan-configs/package.json",
```

Comment:
Nice! I didn't know you can tell greenkeeper where to look for `package.json` files

Reply:
i made this file automatically /shrug